packages/uma-auth-client/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@uma-sdk/uma-auth-client",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/uma-auth-client.umd.cjs",
+  "exports": {
+    ".": {
+      "types": "./dist/uma-auth-client.d.ts",
+      "import": {
+        "types": "./dist/uma-auth-client.d.ts",
+        "default": "./dist/uma-auth-client.js"
+      },
+      "require": {
+        "types": "./dist/uma-auth-client.d.ts",
+        "default": "./dist/uma-auth-client.umd.cjs"
+      }
+    }
+  },
+  "module": "./dist/uma-auth-client.js",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "lint:fix": "eslint --fix .",
+    "preview": "vite preview",
+    "format:fix": "prettier src --write",
+    "format": "prettier src --check"
+  },
+  "dependencies": {
+    "@babel/core": "^7.21.4",
+    "@emotion/css": "^11.11.2",
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@lightsparkdev/ui": "^1.0.3",
```

Comment:
I know we had a whole conversation about this, but can you remind me where we landed? I know we'd said in the past that we didn't want UMA to have any deps on lightspark stuff, so I just want to make sure we're being thoughtful here.